pkg/contracts/src/RegistryCommunityV0_0.sol

```diff
@@ -219,11 +211,13 @@ contract RegistryCommunityV0_0 is
         address[] strategies;
     }
 
-    function initialize(RegistryCommunityV0_0.InitializeParams memory params, address _collateralVaultTemplate)
-        public
-        initializer
-    {
-        __Ownable_init();
```

Comment:
init normally need be called

Reply:
https://github.com/1Hive/gardens-v2/pull/361#discussion_r1726171409